docker-entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Migrate database
+python manage.py migrate
+
+exec $@
```

Comment:
This just execs whatever command you pass into the script, right? If so, I don't see you passing anything into the script in the Dockerfile. I might be missing something here...